contrib/IECoreAppleseed/include/IECoreAppleseed/CameraAlgo.h

```diff
@@ -49,6 +49,13 @@ namespace CameraAlgo
 
 IECOREAPPLESEED_API renderer::Camera *convert( const IECoreScene::Camera *camera );
 
+// This tiny method is only declared here for use in a unit test, because Appleseed doesn't expose
```

Comment:
Looks like it might be available in https://github.com/appleseedhq/appleseed/pull/2202. I suggest writing the test to use that binding, and marking it as expectedFailure until we update to that appleseed version.

Reply:
Oh, yeah, Est added that because I was asking for a better solution to this, forgot to come back and update this.